packages/components/action-button/src/utils/constants.js

```diff
@@ -0,0 +1,11 @@
+export const types = {
+	gray: 'gray',
+	blue: 'blue',
+	inverted: 'inverted'
```

Comment:
I suggest to change the keys to UPPERCASE for consistency with the `ts-button` component types.

Also, I recommend to change the direct names of the colors to the "type" of the `action-button`, the same how it is implemented in the `ts-button` component (e.g. "ACCEPT", "WARNING", "PRIMARY")

Reply:
Uppercase keys are fixed. 

Unfortunately, I have no idea what is the real purpose of color variants of action buttons.
For example, the "inverted" style of the action button used just for dark backgrounds. it's purpose just usage for another theme, not another logical "type" of a button. Type of button is the same - action button, but with little style variants.
I made button types short and equal to colors, to have more brief API and avoid duplication.

It's possible to fix it in two ways:
- Make it more verbose
`<ts-action-button type="**action-**blue"/>`
- Pick attribute `color` instead of `type`. 
 `<ts-action-button color="blue"/>`

Reply:
Maybe @mihneaz  or Vlad can help to find a right name for the type, each style should be related to specific based on our design guideline